consensus/istanbul/backend/backend_test.go

```diff
@@ -53,395 +49,386 @@ var (
 	committeeBlocks = map[Pair]bool{
 		{Sequence: 0, Round: 0}:   false,
 		{Sequence: 0, Round: 1}:   false,
-		{Sequence: 0, Round: 2}:   false,
-		{Sequence: 0, Round: 3}:   false,
-		{Sequence: 0, Round: 4}:   false,
-		{Sequence: 0, Round: 5}:   false,
 		{Sequence: 0, Round: 6}:   false,
-		{Sequence: 0, Round: 7}:   false,
-		{Sequence: 0, Round: 8}:   false,
-		{Sequence: 0, Round: 9}:   false,
 		{Sequence: 0, Round: 10}:  false,
-		{Sequence: 0, Round: 11}:  false,
-		{Sequence: 0, Round: 12}:  false,
-		{Sequence: 0, Round: 13}:  false,
-		{Sequence: 0, Round: 14}:  false,
+		{Sequence: 1, Round: 2}:   false,
```

Comment:
@yoomee1313 What does this `false` mean?
Why are all blocks `false`?

Reply:
@KimKyungup The meaning of `committeeBlocks` is a kind of expect value for the purpose of checking whether the testing node is included as a committee in the specific view{sequence, round}.
During the test, false is changed to true when testing node is included in the specific view. Then, a warning message is printed if any of the elements of the committeeBlocks are not visited.
`committeeBlocks` seems a bit ambiguous. Would it be better to rename it, such as `expectView`?